### DISCUSSION TITLE
iOS build error

### DISCUSSION THREAD

**Discussion by DrunkBunny** [yasirkula/UnityNativeGallery#204] [General]
Hi, I'm using the NativeGallery and it's awesome. However, recently I got the error when build the XCode project with NativeGallery. The errors are attached, can you please take a look? Thanks. ( I'm using Unity 2020.3.6 LTS version, with XCode 12.5.1, the minium iOS target version is 11.0.)
<img width="1157" alt="Screen Shot 2021-06-23 at 4 24 07 PM" src="https://user-images.githubusercontent.com/16040789/123062857-7cfe4980-d43f-11eb-9007-9c9de34c94d7.png">

**Comment by yasirkula** [comment]
Please see: #200

**Reply by DrunkBunny** [reply]
Thank you for the quick reply. I'll check that out:)
